frontend/src/api/apis/plant-controller-api.ts

```diff
@@ -74,6 +74,50 @@ export const PlantControllerApiAxiosParamCreator = function (configuration?: Con
                 options: localVarRequestOptions,
             };
         },
+        /**
```

Comment:
Nie widzę pliku `Forum.tsx`, ale tam w linijce 37 wywaliło błąd. Skoro nie ma tych wartości, to albo na ten moment ma to być zhardkorowane, np. jako stała tablica true/false (jeżeli w przyszłości pole `isFavourite` się pojawi, to się to wtedy poprawi), albo wyrzucić tą funkcjonalność

Reply:
gościnnie generowane, halo #front